proto/ptypes/syncagent.proto

```diff
@@ -65,6 +65,12 @@ message FilesSyncRequest {
   repeated SyncFileInfo sync_file_info_list = 3;
   bool fast_sync = 4;
   int32 file_sync_http_client_timeout = 5;
+  FileLocalSync local_sync = 6;
```

Comment:
NIT: I am thinking we can reuse the existing fields `from_address` and `to_host` for local sync. But a boolean field is required instead. I am fine with the current implementation anyway.